color/src/commonMain/kotlin/ToColor.kt

```diff
@@ -0,0 +1,44 @@
+package com.juul.krayon.color
+
+private const val HEX = "[a-f0-9]"
+
+private val RGB = """($HEX)($HEX)($HEX)""".toRegex(RegexOption.IGNORE_CASE)
+private val RGBA = """($HEX)($HEX)($HEX)($HEX)""".toRegex(RegexOption.IGNORE_CASE)
+private val RRGGBB = """($HEX{2})($HEX{2})($HEX{2})""".toRegex(RegexOption.IGNORE_CASE)
+private val RRGGBBAA = """($HEX{2})($HEX{2})($HEX{2})($HEX{2})""".toRegex(RegexOption.IGNORE_CASE)
```

Comment:
I suspect that regex approach is slower than processing by iterating over the hex characters.
But we can definitely optimize later (and should benchmark to verify that regex is even slower).

Created #25 (super low priority though) to potentially address this later.